cookbooks/bcpc-hadoop/templates/default/hb_hbase-site.xml.erb

```diff
@@ -94,4 +94,71 @@
     <value>/var/run/hadoop-hdfs/dn._PORT</value>
   </property>
 
+  <% if node[:bcpc][:hadoop][:kerberos][:enable] == true then %>
+  <!-- HBase Authorization Configuration -->
+  <property> 
+    <name>hbase.security.authorization</name> 
+    <value>true</value> 
+  </property> 
+
+  <property>    
+    <name>hbase.superuser</name>    
```

Comment:
This potentially breaks the pull request and make hbase not working found by Lev. If build this with new cluster, the hbase will not work, and got some error like :  
```
ERROR: org.apache.hadoop.hbase.security.AccessDeniedException: Insufficient permissions for user 'rwang238' (global, action=CREATE)
```